src/NLog.StructuredLogging.Json/LoggerExtensions.cs

```diff
@@ -35,7 +35,7 @@ public static void ExtendedError(this ILogger logger, string message, object log
         }
 
         public static void Extended(this ILogger logger, LogLevel logLevel, string message, object logProperties,
-            Exception ex = null)
+            Exception ex = null, string tag = "")
```

Comment:
Also, technically this is a breaking change as you'd need to recompile, so would need a version bump if we're going to be strict on semantic versioning.